agent/app/agent.go

```diff
@@ -46,6 +46,8 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	aws_credentials "github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/aws/aws-sdk-go/aws/defaults"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	ecs_service "github.com/aws/aws-sdk-go/service/ecs"
```

Comment:
Please use the private generated client that we have at `github.com/aws/amazon-ecs-agent/agent/ecs_client/model/ecs` instead.  I'd rather not vendor two copies of the client.  We *may* need to update the [generator](https://github.com/aws/amazon-ecs-agent/blob/v1.14.5/agent/gogenerate/awssdk.go) to generate these constants, but I would prefer that to vendoring an additional copy.